arches_lingo/src/arches_lingo/components/scheme/SchemeLabel/components/SchemeLabelViewer.vue

```diff
@@ -0,0 +1,115 @@
+<script setup lang="ts">
+import { inject } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import Button from "primevue/button";
+
+import MetaStringViewer from "@/arches_lingo/components/generic/MetaStringViewer.vue";
+
+import NonLocalizedStringWidget from "@/arches_component_lab/widgets/NonLocalizedStringWidget/NonLocalizedStringWidget.vue";
+import ReferenceSelectWidget from "@/arches_controlled_lists/widgets/ReferenceSelectWidget/ReferenceSelectWidget.vue";
+import ResourceInstanceMultiSelectWidget from "@/arches_component_lab/widgets/ResourceInstanceMultiSelectWidget/ResourceInstanceMultiSelectWidget.vue";
+
+import { VIEW } from "@/arches_lingo/constants.ts";
+
+import type {
+    AppellativeStatus,
+    MetaStringText,
+} from "@/arches_lingo/types.ts";
+
+const props = defineProps<{
+    schemeLabels: AppellativeStatus[];
+}>();
+
+// need NodeGroup info here ( cardinality ) for proper button text
```

Comment:
maybe I'm missing something, but we're already hardcoding cardinality, see L21 where schemeLabels is an array. we can definitely do something for this when building generic forms in provenance.

Reply:
yeah i was hoping for something more generic && expandable, but maybe we skip for now 🤔